### PR TITLE
MES-3668: Health Declaration Category B Structure Refactor

### PR DESCRIPTION
**Pull request by faggc**
## Description
A restructure of the category B files in preparation for the creation of category B+E pages.

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [x] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/health-declaration/cat-b/health-declaration.cat-b.page.ts [hunk 1]
```diff
@@ -56,12 +57,9 @@ interface HealthDeclarationPageState {
 @IonicPage()
 @Component({
   selector: 'page-health-declaration',
```
Comment (appleabs) [inline src/pages/health-declaration/cat-b/health-declaration.cat-b.page.ts]: should be health-declaration-cat-b-page

Comment (faggc) [reply src/pages/health-declaration/cat-b/health-declaration.cat-b.page.ts]: done

File: src/pages/health-declaration/health-declaration.module.ts [hunk 2]
```diff
@@ -1,26 +1,12 @@
...
-import { ComponentsModule } from '../../components/common/common-components.module';
-import { ReactiveFormsModule } from '@angular/forms';
-import { TranslateModule } from 'ng2-translate';
-import { HealthDeclarationEffects } from './health-declaration.effects';
+import { HealthDeclarationCatBPageModule } from './cat-b/health-declaration.cat-b.page.module';
 
 @NgModule({
   declarations: [
-    HealthDeclarationPage,
+    HealthDeclarationCatBPageModule,
```
Comment (appleabs) [inline src/pages/health-declaration/health-declaration.module.ts]: only needed on imports

**Review by TomBillingtonUK — APPROVED** [review]
Looking good :)

**Review by MarkPatt89 — APPROVED** [review]
Nice one!

**Review by appleabs — APPROVED** [review]
nice :)

File: src/pages/health-declaration/health-declaration.module.ts (hunk 2 above)
Comment (faggc) [reply src/pages/health-declaration/health-declaration.module.ts]: done

**Pull request merged**